abpframework/abp
# NgbModalOptions Examples

### Description
I have added two examples demonstrating two frequently used NgbModalOptions properties. Each example is accompanied by visual materials (GIFs/PNG) to clearly illustrate the effects of these properties in action. This update is intended to help developers understand how these properties can be utilised in their applications.
Resolves #18315 (write the related issue number if available)

TODO: 
- Added two new examples showcasing the usage of frequently used NgbModalOptions properties.
- Included GIFs and PNGs for each example to visually demonstrate the modal behavior.
- No breaking changes introduced. 
### Checklist


### How to test it?

- Navigate to the examples section in the documentation.
- Observe the behavior of the modals when the two NgbModalOptions properties are applied.
- Verify that the GIFs/PNGs correctly illustrate the modal’s behavior as described.
